src/postgraphql/graphiql/public/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>GraphiQL</title>
+    <meta name="robots" content="noindex"/>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script>
+      window.POSTGRAPHQL_CONFIG = {
+        graphqlUrl: 'http://localhost:5000/graphql',
+        streamUrl: 'http://localhost:5000/_postgraphql/stream',
```

Comment:
This seems to be taken care of here https://github.com/calebmer/postgraphql/blob/8f93257769258d39dc294072d4d1d84a9cb02094/src/postgraphql/http/createPostGraphQLHttpRequestHandler.js#L95-L98